### DISCUSSION TITLE
Next version?

### DISCUSSION THREAD

**Discussion by Heilemann** [flauwekeul/honeycomb#80] [General]
Hi,

Looks like v4 stalled out about a year ago. Any chance it will see the light of day?

A feature I'd love to see is the ability for it to render a grid of squares as well.

**Comment by flauwekeul** [comment]
Hi,

You're the first one to ask 🥲 Which is part of the reason it got stalled. The biggest part was that I just wasn't satisfied with it and I wanted to do other things for a (long) while. Anyway, I haven't forgotten about it (unfinished projects keep gnawing at me 🐀). I'll see if I can muster up the courage to go at it again.

> A feature I'd love to see is the ability for it to render a grid of squares as well.

That would be a different project, I won't add that to honeycomb. Creating a grid of squares is quite trivial compared to a grid of hexagons. Any specific functionality you're looking for in a library that works with squares?

**Reply by Heilemann** [reply]
So I'm poking at making a tabletop tool, so basically a map with a grid overlay that you can move tokens around on. The sorts of things I'd want to do with it is measuring distances, moving things around on it (snapping to center, lines and stuff like that).

The reason I was thinking about squares in the same package was that way the upper level API stuff, like getting a square/hex from point could be the same.
